libr/util/num.c

```diff
@@ -408,17 +409,54 @@ R_API ut64 r_num_get_input_value(RNum *num, const char *input_value) {
 	return value;
 }
 
+#define NIBBLE_TO_HEX(n) (((n) & 0xf) > 9 ? 'a' + ((n) & 0xf) - 10 : '0' + ((n) & 0xf))
 R_API char* r_num_as_string(RNum *___, ut64 n) {
```

Comment:
Wouldn't it be better to have an additional function (r_num_as_string_ex) to keep the API stable? r_num_as_string could simply call this funtion with default parameters (disabling escapes in this case). Or what's your current approach regarding API changes?